### PR TITLE
Update MathML note

### PR DESCRIPTION
**Pull request by averms**
Chromium-based browsers support MathML as of January 2023. Firefox and Safari already did.

### CODE REVIEW

File: docs/output-formats/html-basics.qmd [hunk 1]
```diff
@@ -149,7 +149,7 @@ Available math rendering methods include:
 | `katex`   | Use [KaTeX](https://github.com/Khan/KaTeX) to display embedded TeX math in HTML output.                                                                                                                                          |
 | `webtex`  | Convert TeX formulas to `<img>` tags that link to an external script that converts formulas to images.                                                                                                                           |
 | `gladtex` | Enclose TeX math in `<eq>` tags in HTML output. The resulting HTML can then be processed by [GladTeX](https://humenda.github.io/GladTeX/) to produce images of the typeset formulas and an HTML file with links to these images. |
-| `mathml`  | Convert TeX math to [MathML](https://www.w3.org/Math/) (note that currently only Firefox and Safari natively support MathML)                                                                                                     |
+| `mathml`  | Convert TeX math to [MathML](https://www.w3.org/Math/).
```
Comment (mcanouil) [inline docs/output-formats/html-basics.qmd:152]: You broke the table.
Could you fix it?

**Comment by mcanouil** [comment]
Do you have a source saying that it is supported?

File: docs/output-formats/html-basics.qmd (hunk 1 above)
Comment (averms) [reply docs/output-formats/html-basics.qmd]: Oops sorry

**Comment by averms** [comment]
Just force-pushed commits with a fixed table.

> Do you have a source saying that it is supported?

The Chrome 109 release notes mention it: https://developer.chrome.com/blog/new-in-chrome-109#math-ml-core and I found a blog post from the company that contributed support: https://www.igalia.com/2023/01/10/Igalia-Brings-MathML-Back-to-Chromium.html.

**Pull request merged**